docs/introduction/faq.md

```diff
@@ -0,0 +1,95 @@
+# FAQ
+
+## Overview
+
+### How do I get Atoms?
+
+If you participated in the fundraiser, you can check your suggested atom balance at [fundraiser.cosmos.network](https://fundraiser.cosmos.network).
+If not, you must wait until the [Cosmos Network launches](https://cosmos.network/roadmap) and Atoms are traded on exchanges.
+
+### Are Atoms listed on exchanges?
+
+No. The Cosmos Network mainnet has not yet launched, which means Atoms are _not_ on exchanges yet. $CMOS and $ATOM tokens are _not_ Cosmos Network native tokens.
+
+### How do I participate in the fundraiser?
+
+The [fundraiser](https://fundraiser.cosmos.network) is closed. The Interchain Foundation raised funds from private individuals and has hosted a public fundraising event on which ended on April 6, 2017. Both $ETH and $BTC were accepted in the fundraiser. The security of the fundraising process has been vetted extremely carefully.
+
+### What is the initial allocation of Atoms?
+
+As a public, decentralized network, the allocation of Atoms is decided by those who run the software for the Cosmos Hub. To faciliate a decision, we are creating a Swiss non-profit, the [Interchain Foundation](https://interchain.io), which is responsible for co-ordinating fundraising and allocating funds to get the network off the ground. The foundation will suggest a allocation of Atoms according to the results of the fundraiser. Users will ultimately decide the distribution for themselves when they run the software.
```

Comment:
Maybe 

"As a public, decentralized network, the allocation of Atoms is decided by those who run the software for the Cosmos Hub. To this end, we have created a Swiss non-profit, the [Interchain Foundation](https://interchain.io) which is responsible for helping build and grow the Cosmos network. The foundation will suggest an allocation of Atoms according to the results of the fundraiser."